src/decoder/vorbis.rs

```diff
@@ -133,13 +147,8 @@ fn is_vorbis<R>(mut data: R) -> bool
 where
     R: Read + Seek,
 {
-    let stream_pos = data.stream_position().unwrap();
-
-    if OggStreamReader::new(data.by_ref()).is_err() {
-        data.seek(SeekFrom::Start(stream_pos)).unwrap();
-        return false;
-    }
-
-    data.seek(SeekFrom::Start(stream_pos)).unwrap();
-    true
+    let stream_pos = data.stream_position().unwrap_or_default();
+    let result = OggStreamReader::new(data.by_ref()).is_ok();
+    let _ = data.seek(SeekFrom::Start(stream_pos));
```

Comment:
Why are such errors ignored now?

Reply:
The intent is to decrease the probability of panicking. 

When the stream position could not be gotten or restored, then old behavior was to panic.

New behavior is to still probe whether a demuxer/decoder can be initialized (whether we "are Vorbis, MP3, ...") and continue. This may still trigger the old behavior, because if the stream cannot be rewound, then in the `new` method that calls this `is_vorbis` it is likely that re-initializing the decoder will fail on this advance position.